core/integration/features/spot/orders/0037-OPEG-023.feature

```diff
@@ -0,0 +1,79 @@
+      | market.liquidity.earlyExitPenalty                   | 0.25  |
+      | market.liquidity.stakeToCcyVolume                   | 1.0   |
+      | market.liquidity.sla.nonPerformanceBondPenaltySlope | 0.19  |
+      | market.liquidity.sla.nonPerformanceBondPenaltyMax   | 1     |
+
+    And the liquidity monitoring parameters:
+      | name       | triggering ratio | time window | scaling factor |
+      | lqm-params | 0.1              | 24h         | 1              |
+
+    And the following assets are registered:
+      | id  | decimal places |
+      | ETH | 1              |
+
+    And the average block duration is "1"
+    And the simple risk model named "simple-risk-model-1":
+      | long | short | max move up | min move down | probability of trading |
+      | 0.1  | 0.1   | 60          | 50            | 0.2                    |
+    And the fees configuration named "fees-config-1":
+      | maker fee | infrastructure fee |
+      | 0.004     | 0.001              |
+    And the price monitoring named "price-monitoring-1":
+      | horizon | probability | auction extension |
+      | 1       | 0.99        | 5                 |
+    And the liquidity sla params named "SLA":
+      | price range | commitment min time fraction | performance hysteresis epochs | sla competition factor |
+      | 0.01        | 0.5                          | 1                             | 1.0                    |
+
+  Scenario:
+    #0037-OPEG-023:Given a market with non-zero market and asset decimals where the asset decimals are strictly less than the market decimals (yielding a negative price factor). A pegged order specifying an offset which is not an integer multiple of the markets tick size should be rejected.
+    #0037-OPEG-024:Given a market with non-zero market and asset decimals where the asset decimals are equal to the market decimals (yielding a negative price factor). A pegged order specifying an offset which is not an integer multiple of the markets tick size should be rejected.
+    #0037-OPEG-027:Given a market with non-zero market and asset decimals where the asset decimals are strictly more than the market decimals (yielding a negative price factor). A pegged order specifying an offset which is not an integer multiple of the markets tick size should be rejected.
+    And the spot markets:
+      | id        | name    | quote asset | base asset | liquidity monitoring | risk model          | auction duration | fees          | price monitoring   | sla params    | decimal places | tick size |
+      | ETH/DEC21 | BTC/ETH | BTC         | ETH        | lqm-params           | simple-risk-model-1 | 1                | fees-config-1 | price-monitoring-1 | default-basic | 2              | 10        |
+      | ETH/DEC22 | BTC/ETH | BTC         | ETH        | lqm-params           | simple-risk-model-1 | 1                | fees-config-1 | price-monitoring-1 | default-basic | 2              | 5         |
```

Comment:
Maybe this will become apparent once the test is updated to get out of opening auction, but these markets shouldn't be possible right?

We have...
- `asset decimals = 1`
- `market decimals = 2`
- `position decimals = 0`

Therefore violating the required `market decimal + position decimal < asset decimals`